### ISSUE TITLE
[BUG] [Python] UNKNOWNBASETYPE appear in api py 

### ISSUE THREAD

**Issue by DoneSpeak** [OpenAPITools/openapi-generator#12404]
#### Bug Report Checklist

- [ ] Have you provided a full/minimal spec to reproduce the issue?
- [ ] Have you validated the input using an OpenAPI validator ([example](https://apidevtools.org/swagger-parser/online/))?
- [x] Have you [tested with the latest master](https://github.com/OpenAPITools/openapi-generator/wiki/FAQ#how-to-test-with-the-latest-master-of-openapi-generator) to confirm the issue still exists?
- [x] Have you searched for related issues/PRs?
- [ ] What's the actual output vs expected output?
- [ ] [Optional] Sponsorship to speed up the bug fix or feature request ([example](https://github.com/OpenAPITools/openapi-generator/issues/6178))

##### openapi-generator version

master

##### OpenAPI declaration file content or url

[swagger.json](https://github.com/OpenAPITools/openapi-generator/files/8735171/swagger.json.txt)

##### Generation Details

```
> python3 --version
Python 3.8.10

> java --version                                                                                            
openjdk 11.0.14 2022-01-18 LTS
OpenJDK Runtime Environment Zulu11.54+23-CA (build 11.0.14+9-LTS)
OpenJDK 64-Bit Server VM Zulu11.54+23-CA (build 11.0.14+9-LTS, mixed mode)
```

```
git clone https://github.com/openapitools/openapi-generator
cd openapi-generator
mvn clean package
java -jar modules/openapi-generator-cli/target/openapi-generator-cli.jar  generate \
    -i api-def/swagger.json \
    -g python \
    --artifact-id symbility_api \
    --package-name symbility_api \
    -o api/
```

##### Steps to reproduce

see `ap/symbility_api/api/claim_api.py`

The `unknown_base_type`, `UNKNOWNBASETYPE`, `unknownbasetype` and `UNKNOWN_BASE_TYPE` donesn't exist.

```python
from symbility_api.model.unknownbasetype import UNKNOWNBASETYPE
.....
        self.claim_set_claim_status_endpoint = _Endpoint(
            settings={
                'response_type': (ClaimStatus,),
                'auth': [
                    'OAuth2'
                ],
                'endpoint_path': '/v03_08/claims/{claimID}/claim-status',
                'operation_id': 'claim_set_claim_status',
                'http_method': 'PUT',
                'servers': None,
            },
            params_map={
                'all': [
                    'claim_id',
                    'from_user_id_specification',
                    'unknown_base_type',
                ],
                'required': [
                    'claim_id',
                ],
                'nullable': [
                    'unknown_base_type',
                ],
                'enum': [
                ],
                'validation': [
                ]
            },
            root_map={
                'validations': {
                },
                'allowed_values': {
                },
                'openapi_types': {
                    'claim_id':
                        (str,),
                    'from_user_id_specification':
                        (str,),
                    'unknown_base_type':
                        (UNKNOWN_BASE_TYPE,),
                },
                'attribute_map': {
                    'claim_id': 'claimID',
                    'from_user_id_specification': 'From-User-ID-Specification',
                },
                'location_map': {
                    'claim_id': 'path',
                    'from_user_id_specification': 'header',
                    'unknown_base_type': 'body',
                },
                'collection_format_map': {
                }
            },
            headers_map={
                'accept': [
                    'application/json'
                ],
                'content_type': [
                    'application/json'
                ]
            },
            api_client=api_client
        )
```

after remove the unknowtype, I got:

![image](https://user-images.githubusercontent.com/16522248/169433515-ef6ba526-086a-491a-9327-8eb8c46d94a5.png)

![image](https://user-images.githubusercontent.com/16522248/169433529-8ae3cb3b-b11b-4c51-92ac-94005c8a6334.png)

**Comment by wing328** [comment]
Thanks for reporting the issue. Can you try the branch `inline-parameter-enhance` in https://github.com/OpenAPITools/openapi-generator/pull/12369, which provides better support for inline schemas?

**Comment by wing328** [comment]
The fix has been merged into master and included in the v6.0.0 release.
